definition/IRoom.ts

```diff
@@ -32,6 +32,7 @@ export interface IRoom extends IRocketChatRecord {
 	lm?: Date;
 	usersCount: number;
 	jitsiTimeout: Date;
+	callStatus: string;
```

Comment:
```suggestion
	callStatus?: string;
```
This doesn't need to be required.

Reply:
Also since the callStatus can only have finite states, why not create an explicit type for it rather that just assigning a general type of `string`